### PR TITLE
New density map module

### PR DESCRIPTION
**Pull request by GuillaumeTh**
## Describe your changes

New module to compute a density map from a bundle.

## List test packages used by your module

bundle.zip

## Checklist before requesting a review

- Create the tool:
  - [x] Edit `./modules/nf-neuro/<category>/<tool>/main.nf`
  - [x] Edit `./modules/nf-neuro/<category>/<tool>/meta.yml`
  - [x] Edit `./modules/nf-neuro/<category>/<tool>/environment.yml`
- Generate the tests:
  - [x] Edit `./modules/nf-neuro/<category>/<tool>/tests/main.nf.test`
  - [x] Run the tests to generate the `main.nf.test.snap` snapshots
- Ensure the syntax is correct :
  - [x] Run `prettier` and `editorconfig-checker` to fix common syntax issues
  - [x] Run `nf-core modules lint` and fix all errors
  - [x] Ensure your variables have good, clear names

### CODE REVIEW

File: modules/nf-neuro/bundle/densitymap/meta.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/scilus/nf-neuro/main/modules/meta-schema.json
+name: "bundle_densitymap"
+description: Compute density maps from tractograms
+keywords:
+  - Diffusion MRI
+  - Tractogram
+  - Density Map
+tools:
+  - "Scilpy":
```
Comment (arnaudbore) [inline modules/nf-neuro/bundle/densitymap/meta.yml:9]:
```suggestion
  - "scilpy":
```

**Review by arnaudbore — APPROVED** [review]
Quick modification and LGTM

File: modules/nf-neuro/tractogram/densitymap/main.nf [hunk 2]
```diff
@@ -0,0 +1,46 @@
+process TRACTOGRAM_DENSITYMAP {
+    tag "$meta.id"
+    label 'process_single'
+
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://scil.usherbrooke.ca/containers/scilus_2.0.2.sif':
+        'scilus/scilus:2.0.2' }"
+
+    input:
+    tuple val(meta), path(bundle)
```
Comment (arnaudbore) [inline modules/nf-neuro/tractogram/densitymap/main.nf:10]:
```suggestion
    tuple val(meta), path(tractogram)
```

File: modules/nf-neuro/tractogram/densitymap/meta.yml [hunk 3]
```diff
@@ -0,0 +1,45 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/scilus/nf-neuro/main/modules/meta-schema.json
```
Comment (arnaudbore) [inline modules/nf-neuro/tractogram/densitymap/meta.yml:1]:
```suggestion
---
```

**Review by arnaudbore — APPROVED** [review]

**Review by ThoumyreStanislas — APPROVED** [review]
LGTM

**Pull request merged**